lxc/linuxcontainers.org
# Replace Bootstrap with Vanilla as the CSS framework

## Done
Replaced Bootstrap with Vanilla as the CSS framework. Integrated the framework by replacing the components and updating class names.

## QA
- This branch will require an entire site check
- Check the try it page
- Check the different languages work
- Check on small screens

## Details
Fixes https://github.com/ubuntudesign/vanilla-squad/issues/103
